plugins/vertica/database_credentials.go

```diff
@@ -0,0 +1,55 @@
+package vertica
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func DatabaseCredentials() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.DatabaseCredentials,
+		DocsURL:       sdk.URL("https://www.vertica.com/docs/9.2.x/HTML/Content/Authoring/ConnectingToVertica/vsql/Install/InstallingTheVsqlClient.htm"), 
+		ManagementURL: sdk.URL("https://www.vertica.com/try/"), 
+		Fields: []schema.CredentialField{
+			{
+				Name:                fieldname.Host,
+				MarkdownDescription: "Vertica host to connect to.",
+				Optional:            true,
+			},
+			{
+				Name:                fieldname.Port,
+				MarkdownDescription: "Port used to connect to Vertica.",
+				Optional:            true,
+			},
+			{
+				Name:                fieldname.Username,
+				MarkdownDescription: "Vertica user to authenticate as.",
+				Optional:            true,
+			},
```

Comment:
I see everything except for the password is marked as optional. Is there some documentation related to the default behaviour, when the rest of the values are not provisioned?

Reply:
by default it will use localhost and default creds.
But i have added all creds as required as it is used mostly for cloud which requires all creds to explicitly.

Reply:
Hmm, I think I preferred the prior version better - marking them as optional makes this plugin harder to use for folks just wanting to give it a try on localhost, before transitioning their prod workflows to Shell Plugins.